test/stop_containers.sh

```diff
@@ -4,11 +4,8 @@ set -eou pipefail
 
 export WEAVIATE_VERSION=$1
 
-docker compose -f test/docker-compose.yml down --remove-orphans
-docker compose -f test/docker-compose-azure.yml down --remove-orphans
-docker compose -f test/docker-compose-okta-cc.yml down --remove-orphans
-docker compose -f test/docker-compose-okta-users.yml down --remove-orphans
-docker compose -f test/docker-compose-wcs.yml down --remove-orphans
-docker compose -f test/docker-compose-cluster.yml down --remove-orphans
+# Removes containers in the default docker-compose.yml
+# and _any other containers_ not defined in that file.
+docker compose down --remove-orphans
```

Comment:
question: this will tear down all of the docker compose setups?

Reply:
Yes, I tried locally with starting several "groups" of containers -- running this command once removes all of them.

Because it's `remove all containers in docker-compose.yml` + `remove any container that's not in the docker-compose.yml` -- effectively **all** containers